tests/index.test.ts

```diff
@@ -131,6 +171,21 @@ describe("CLI", () => {
     );
     expect(collect).toHaveBeenCalledTimes(2);
     expect(upload).toHaveBeenCalledTimes(2);
+
+    expect(preTranslate).not.toHaveBeenCalled();
```

Comment:
NBC: I think this would be better in a separate test so it can be ran in parallel